src/components/Carousel/Carousel.tsx

```diff
@@ -642,7 +642,7 @@ export const Carousel: FC<CarouselProps> = React.forwardRef(
                     onCurrentChange={(currentPage: number) =>
                       handleIndicatorClick(currentPage - 1)
                     }
-                    pageSizes={[1]}
+                    pageSize={1}
```

Comment:
You'll need to ensure this change works both with `pageSizes` and `pageSize` props to ensure no existing implementations are broken.

Reply:
This specific change is harmless as its a change within the library. But I get the gist of the comment.

Reply:
Tested both using `Carousel.tsx` and LGTM